src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,193 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import subway.line.LineResponse;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 노선 관련 기능")
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    private static final String BASE_URL = "/lines";
+    private static final String STATION_URL = "/stations";
+    private Station station1, station2, station3;
+
+    @BeforeEach
+    void setUp() {
+        station1 = createStation("강남역");
+        station2 = createStation("역삼역");
+        station3 = createStation("양재역");
```

Comment:
1,2,3 처럼 변수명을 넘버링 하는 습관은 좋지 않아요, 아래와 같이 변수명에 의미를 담으면 좋을 것 같습니다.
```suggestion
        강남역 = createStation("강남역");
        역삼역 = createStation("역삼역");
        양재역 = createStation("양재역");
```